packages/core/helper-plugin/lib/src/components/SearchURLQuery/index.js

```diff
@@ -38,7 +38,11 @@ const SearchURLQuery = ({ label, placeholder, trackedEvent }) => {
 
     if (value) {
       if (trackedEvent) {
-        trackUsage(trackedEvent);
+        if (trackedEventDetails) {
+          trackUsage(trackedEvent, trackedEventDetails);
```

Comment:
I wondered about creating 1 object handling both event name + event eventual details but I know we should be careful about updating helper-plugin components 🤔